### ISSUE TITLE
Planning Your s2Member Configuration

### ISSUE THREAD

**Issue by patdumond** [websharks/s2member-kb#297]
##### KB Article Creation Checklist

- [x] Write initial draft for this KB Article; label this issue `draft` and `s2guide`
- [x] Add [required YAML configuration](https://github.com/websharks/team/wiki/KB-Articles-::-Required-YAML-Configuration)
- [x] Add Tags for this KB Article to the YAML config (see [YAML Keys (Explained)](https://github.com/websharks/team/wiki/KB-Articles-::-Required-YAML-Configuration#yaml-keys-explained))
- [x] Edit and finalize draft for publishing (remove `draft` label, add `draft-finalized` label)
- [x] Assign Issue to yourself and [create Markdown file](https://github.com/websharks/team/wiki/KB-Articles-::-Creating-the-Markdown-File) (remove `draft-finalized` label, add `pending`)
- [x] Project Lead: Review and Publish KB Article (remove `pending` label, add `published` label)

---

### KB Article Published @ s2member.com<br />:page_with_curl: See: [Planning Your s2Member Configuration](http://s2member.com/kb-article/planning-your-s2member-configuration/)
[:octocat: View Markdown File](https://github.com/websharks/s2member-kb/blob/master/tutorials/2016/297-planning-your-s2member-configuration.md) | [:pencil2: Edit Markdown File](https://github.com/websharks/s2member-kb/edit/master/tutorials/2016/297-planning-your-s2member-configuration.md)

---

**Comment by patdumond** [comment]
@raamdev, @jaswsinc: First draft ready for review.

**Comment by jaswsinc** [comment]
@patdumond Very very nice. I found only a very few minor details in this worth mentioning. Otherwise it looks ready to me. I'll list what I did find below. Thanks! ;-)

> you probably have content on your website that you want to restrict to site members.

- [ ] Suggest title case in the word Members and maybe rewording this slightly to improve clarity just a bit; e.g., "you probably have content on your website that you only want Members to see."

> access content that is restricted above their user level

- [ ] Suggest: "access content that is restricted above their Membership Level"

> WordPress Dashboard →s2Member®

- [ ] There are a few places in the article where I noticed a missing space after `→`. I suggest adding a space after the initial `→` symbol, that way there are no problems whenever s2Member.com parses those into pretty Dashboard paths on the site.

> even if you've set Anyone Can Register to Off in your WordPress General Settings options

- [ ] You could probably exclude the word "options" in this sentence.

> but you can call it Fred if you want -- just remember what you titled it.
> but you can call it Barney if you want -- just remember what you titled it.

:laughing: Hilarious! Love it.

**Comment by patdumond** [comment]
@jaswsinc & @raamdev: I've made the changes indicated by @jaswsinc and also checked for the correct terminology IAW this document: https://github.com/websharks/team/wiki/Industry-Terms-Used-or-Created-by-WS. title case for *Posts*, *Pages*, *Users* *Members*, and *Visitors*. I've also corrected all the URLs.

I'd appreciate it if this could be reviewed quickly as, again, I am a little anxious to start seeing these things published. ;)

**Comment by jaswsinc** [comment]
Thanks Pat!

- 1600 words.
- I'll let Elizabeth know about this.
- It will be published sometime later today.

Keep em' coming :-)

**Comment by patdumond** [comment]
@jaswsinc and @raamdev: Removed *s2Member User's Guide* from title and removed paragraph referencing the *s2Member User's Guide*.

**Comment by jaswsinc** [comment]
Copy that. Thanks! :-)

**Comment by jaswsinc** [comment]
@raamdev Can I publish this now? Or do you want to do that?

**Comment by raamdev** [comment]
@jaswsinc This is just missing [a few things](https://github.com/websharks/s2member-kb/issues/296#issuecomment-209881610). Otherwise, I've reviewed it and it looks great to me!

@patdumond Nice work! :-) I left a comment [here](https://github.com/websharks/s2member-kb/issues/296#issuecomment-209881610) that also applies to this article.

**Comment by patdumond** [comment]
@raamdev: Made the following changes:
- Added User's Guide paragraph at end of introduction
 - Updated YAML title
 - Tagged: membership-levels, login-welcome-page, membership-options-page, restriction-options, pro-forms, login-registration

**Comment by jaswsinc** [comment]
Woohoo! Thanks @patdumond @raamdev ~ I love how this ended up.
